File: TeacherShare/class1/Student.cs

using  System.IO;
using  System;

namespace c208
{
 public class Student
{
   private string id;
   private string name;
   private double score;

   public  void getData(string i,string n,double s)
  {
         id=i;
         name=n;
         score=s;
   }

   public  void putData()
  {  
        Console.WriteLine("id="+id);
        Console.WriteLine("name="+name);
        Console.WriteLine("score="+score);
   } 
    
   public static void Main(string [] args)
  {
    Student s1=new Student();
    Student s2=new Student();
    s1.getData("01","ÕÅÈý·á",585.88);
    s2.getData("02","»ôÔª¼×",555.88);
    s1.putData();
    s2.putData();
    }
}

}

File: TeacherShare/class1/Student.cpp
#include "iostream.h"
#include "math.h"

class Student
{
   private:
     char *id;
     char *name;
     double score;
 
    public:
      void getdata(char *i,char *n,double s);
      void putdata();
};

void main()
{
    Student s1;//给类Student新建一个对象s1;为s1分配内存;调用默认构造函数为s1初始化
    Student s2;
    s1.getdata("01","张三丰",585.88);
    s2.getdata("02","霍元甲",555.88);
    s1.putdata();
    s2.putdata();
}

      void Student::getdata(char *i,char *n,double s)
      {
         id=i;
         name=n;
         score=s;
      }

     void Student::putdata()
     {  
        cout << "id="  << id  << "\n";
        cout << "name="  << name  << endl;        
        cout << "score="  << score  << "\n";
      } 



